.github/workflows/config.yml

```diff
@@ -0,0 +1,51 @@
+name: config
+
+on:
+  push:
+    branches:
+      - master
+      - test
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./node_modules
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run: npm install --ignore-engines
+      - run: npm run build
+      # - uses: actions/upload-artifact@v2
+      #   with:
+      #     name: node_modules
+      #     path: ./node_modules
+      - uses: actions/cache@v2
```

Comment:
```suggestion
      - uses: actions/cache@v3
```

Reply:
@Pavan-Nambi actually this step can be removed because you already have a cache definition at the beginning, right before it, and that's enough.